Examples/Widgets/Notes/package-lock.json

```diff
@@ -0,0 +1,18731 @@
+{
```

Comment:
We dont have to check in package-lock.json into repo

Reply:
.gitignore configuration updated.
package-lock.json removed from repo